requirements/common.txt

```diff
@@ -13,20 +10,15 @@ Flask-Script==2.0.5
 Flask-SQLAlchemy==2.0
 Flask-SSLify==0.1.5
 Flask-WTF==0.11
-geopy==1.11.0
```

Comment:
Oh is it? Ok, will add back. It was previously only in dev.txt, so I thought it was added by accident.